### PR TITLE
fix #76331 kaggle.com

### LINKED ISSUE #76331: kaggle.com
### Issue URL (Annoyance)
 [https://www.kaggle.com/c/titanic/data](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.kaggle.com%2Fc%2Ftitanic%2Fdata)
### Comment
Username: `@Rakleed`
> 
> --
> The information below is automatically generated. Do not delete it.
> 
> Referrer:
> 
> 
> Custom userscripts:
> https://userscripts.adtidy.org/release/adguard-extra/1.0/adguard-extra.user.js
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.5.3
Browser: | Chrome
AdGuard driver: | WFP
Stealth mode options: | Hide your search queries, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters, <br>Self-destructing third-party cookies (180), <br>Hide your Referrer from third-parties (www.kaggle.com), <br>Hide your User-Agent ()
DNS filtering: | server: `https://dns.cloudflare.com/dns-query`<br>
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/> AdGuard Russian

### PR DESCRIPTION
**Pull request by wout-codes**
#76331

### CODE REVIEW

**Comment by Alex-302** [comment]
please do not open new PRs(at least big), as I asked yesterday. All current must be processed

**Comment by wout-codes** [comment]
> please do not open new PRs(at least big), as I asked yesterday. All current must be processed

Okay. I will not open anymore today.

File: AnnoyancesFilter/sections/cookies_specific.txt [hunk 1]
```diff
@@ -92,6 +92,7 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 !
 !
 rshb.ru###alert.bottom
+kaggle.com###site-content ~ div
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/cookies_specific.txt:95]: better `+` instead of `~`, less wide.

Comment (wout-codes) [reply AnnoyancesFilter/sections/cookies_specific.txt:95]: Okay.

**Pull request merged**